src/streaming/controllers/ScheduleController.js

```diff
@@ -384,11 +385,14 @@ function ScheduleController(config) {
                 timelineConverter.setTimeSyncCompleted(true);
                 setLiveEdgeSeekTarget();
             } else {
-                seekTarget = playbackController.getStreamStartTime(false);
                 streamProcessor.getBufferController().setSeekStartTime(seekTarget);
             }
         }
 
+        if (!playbackController.getIsDynamic()) {
```

Comment:
why do we, in static stream use case, have to update the seek start time of the bufferController before the seekTarget?

Reply:
Need to check with the team. I will get back to you tomorrow.

Reply:
That was moved, it really need to be before setSeekStartTime. Fixed and commited.